debezium-server-kinesis/src/main/java/io/debezium/server/kinesis/KinesisChangeConsumer.java

```diff
@@ -78,8 +78,7 @@ void connect() {
         region = config.getValue(PROP_REGION_NAME, String.class);
         endpointOverride = config.getOptionalValue(PROP_ENDPOINT_NAME, String.class);
         final KinesisClientBuilder builder = KinesisClient.builder()
-                .region(Region.of(region))
-                .credentialsProvider(ProfileCredentialsProvider.create(credentialsProfile));
+                .region(Region.of(region));
```

Comment:
With the removal of `ProfileCredentialsProvider` usage here, I suspect the import above is not used, could the unused imports also be removed as well?

Reply:
Yes, we probably can remove the unused imports. I will commit it again.